assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -92,6 +92,7 @@ const Footer = styled.div`
   font-size: ${CONTROLS_WIDTH}px;
   line-height: 19px;
   position: relative;
+  margin-top: 7px;
```

Comment:
@barklund Apparently the issue came just from the footer partially being on top of the slider pointers...